ui/apps/platform/src/Containers/Violations/ViolationsTablePanel.tsx

```diff
@@ -13,13 +13,14 @@ import {
 import { TableComposable, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
 
 import useTableSelection from 'hooks/useTableSelection';
-import { TableColumn, SortDirection } from 'hooks/useTableSort';
+import { TableColumn } from 'hooks/useTableSort';
```

Comment:
we can probably pull this out into a separate table types file

Reply:
Looking through some related areas I found some other stuff that could be moved to a table types file, I'll open up a follow up PR for moving those over.